### PR TITLE
Fixes #14146 - ActivationKey field headers modified for clarity.

### PR DESCRIPTION
**Pull request by zjhuntin**
You can check the changes in:
- Content -> Activation Keys
  - changed "Consumed" to "Host Limit"

- Content -> Activation Keys -> [key] -> Details
  - changed "Content Host Limit" to "Host Limit"

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/views/activation-keys-table-full.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <thead>
     <tr bst-table-head>
       <th bst-table-column="name" sortable><span translate>Name</span></th>
-      <th bst-table-column="consumed"><span translate>Consumed</span></th>
+      <th bst-table-column="Host Limit"><span translate>Consumed</span></th>
```
Comment (cfouant) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/views/activation-keys-table-full.html]: I think you want this:
`<th bst-table-column="consumed"><span translate>Host Limit</span></th>`

Comment (zjhuntin) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/views/activation-keys-table-full.html]: Yup, sorry about that. Thanks for catching it!

**Comment by bbuckingham** [comment]
ACK.

We can merge once https://github.com/Katello/hammer-cli-katello/pull/363 is also ACKed.

**Comment by bbuckingham** [comment]
Merging!

Thanks @zjhuntin !

**Pull request merged**